src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -95,9 +103,26 @@ public void update(String email, User user) {
 
     @Override
     public void setUserStatusTrue(long id) {
-        MapSqlParameterSource sqlParameterSource = new MapSqlParameterSource();
-        sqlParameterSource.addValue("id", id);
+        MapSqlParameterSource sqlParameterSource = new MapSqlParameterSource()
+                .addValue("id", id);
         namedParameterJdbcTemplate.update(UPDATE_USER_STATUS, sqlParameterSource);
     }
 
+    @Override
+    public void changeProfile(User user, String email, String discogsUserName) {
```

Comment:
>    @Override
    public void changeProfile(User user, String email, String discogsUserName) {
        MapSqlParameterSource sqlParameterSource = new MapSqlParameterSource()
                .addValue("new_email", email)
                .addValue("discogs_user_name", discogsUserName)
                .addValue("email", user.getEmail());
        jdbcTemplate.update(UPDATE_PROFILE_FIELDS, sqlParameterSource);
    }

Reply:
What the issue is supposed to be here?